hazelcast/src/main/resources/hazelcast-full-example.yaml

```diff
@@ -3260,6 +3266,10 @@ hazelcast:
       public-address: dummy
       # If the address should be reused. See NetworkConfig.setReuseAddress for more information.
       reuse-address: true
+      alto-socket:
```

Comment:
Similar to XML file, a bit of explanation would be good

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/034070382c199cea19e7f577912ba529eaf055f4